src/dataslice.js

```diff
@@ -83,18 +83,20 @@ export default class DataSlice {
     const right = this.readUint32(offset + 4);
     let combined;
     if (this._littleEndian) {
-      combined = left + 2 ** 32 * right;
+      combined = left + ((2 ** 32) * right);
```

Comment:
The [`no-mixed-operators`](https://eslint.org/docs/rules/no-mixed-operators) rule wants explicit grouping here and below so people don't have to remember precedence rules.